app/subscriber/src/features/search-page/components/advanced-search/components/sections/SearchInGroup.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { useContent } from 'store/hooks';
-import { Checkbox, Col, Row } from 'tno-core';
+import { Col, Row, SmallCheckbox } from 'tno-core';
```

Comment:
Why introduce a new component which is a duplicate of the Checkbox component with styling changes?  Just override the style.  Having a copy will result in future bugs in one or the other.  Or am I missing something?

Reply:
I will remove this. thanks

Reply:
I cannot modify the size of the checkbox with only style changes.

Reply:
updated the Checkbox to be able to take size property.